sdk/iot/azure-iot-modelsrepository/README.md

```diff
@@ -0,0 +1,16 @@
+# Azure IoT Models Repository Library
+
+The Azure IoT Models Repository Library for Python provides functionality for working with the Azure IoT Models Repository
+
+## Installation
+
+This package is not yet available on pip. Please install locally from source:
```

Comment:
Are you planning to update the installation instructions since we aim to release a package on PyPI?

Reply:
I had intended to update this after the package was released, but I can go ahead and do it now.